src/dialog/dialog.tsx

```diff
@@ -1,7 +1,6 @@
-import * as React from 'react';
-
 import { Button } from '@ampel-ui/button';
 import { onOuterClick } from '@ampel-ui/subscriptions';
+import * as React from 'react';
```

Comment:
Do these changes have an intention?

Reply:
auto-formatting i have reverted the changes